docs/enroute/Brisbane Centre/ARL.md

```diff
@@ -91,8 +91,8 @@ When **CFS ADC** is online, **INL** and **MNN** owns the Class C airspace from A
 Departures from YCFS in to MNN Class C will be coordinated when ready for departure.
 
 !!! example
-    <span class="hotline">**CFS ADC** -> **MNN**</span>: "Next, CFH21"  
-    <span class="hotline">**MNN** -> **CFS ADC**</span>: "CFH21"  
+    <span class="coldline">**CFS ADC** -> **MNN**</span>: "Next, CFH21"  
```

Comment:
Thoughts on keeping this hotline?  My thinking was that the activation of the FDR was a heads up, the next call is quick and easy prompt to the next controller.

Reply:
By that logic, everything is hotline :D